products/eclipse-jetty.md

```diff
@@ -0,0 +1,120 @@
+title: Eclipse Jetty
+category: server-app
+tags: eclipse web-server
+iconSlug: eclipsejetty
+permalink: /eclipse-jetty
+releasePolicyLink: https://eclipse.dev/jetty/download.php
+changelogTemplate: https://github.com/jetty/jetty.project/releases/tag/jetty-__LATEST__
+releaseDateColumn: true
+eolColumn: "Community Support"
+extendedSupportColumn: "Extended Support"
+
+identifiers:
+-   purl: pkg:maven/org.eclipse.jetty/jetty-server
+
+auto:
+-   maven: org.eclipse.jetty/jetty-server  
+
+# eol status and minJavaVersion can be found on https://eclipse.dev/jetty/download.php
+releases:
+-   releaseCycle: "12"
+    releaseDate: 2023-08-07
+    eol: false
+    extendedSupport: true
+    minJavaVersion: 17
+    latest: "12.0.3"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/10485
+-   releaseCycle: "11"
+    releaseDate: 2020-12-07
+    eol: 2024-01-01
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "11.0.18"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/10485
+-   releaseCycle: "10"
+    releaseDate: 2020-12-07
+    eol: 2024-01-01
+    extendedSupport: true
+    minJavaVersion: 11
+    latest: "10.0.18"
+    latestReleaseDate: 2023-10-30
+
+# https://github.com/jetty/jetty.project/issues/7958
+-   releaseCycle: "9.4"
+    releaseDate: 2016-12-07
+    eol: 2022-06-01
+    extendedSupport: true
+    minJavaVersion: 8
+    latest: "9.4.53.v20231009"
+    latestReleaseDate: 2023-10-10
+
+-   releaseCycle: "9.3"
+    releaseDate: 2015-06-01
+    eol: 2022-12-31
+    extendedSupport: 2021-10-01
```

Comment:
It's strange to have extended support ending before eol. Is it OK ?

Reply:
Correct, but maybe we better put it on `false`, since it is unclear / unknown.

We agreed to put the end-of-life date based on the public downloads page, which doesn't stipulate a specific date, but we took 31/12 of that year to have an EOL date. For the recent release (9.4 and later) there is a GitHub issue which documents the "end of community support", which I considered as the "end-of-life" date here, and the effective date on the downloads page as the "end of extended support".

I've changed it in the merge request to `false` for those we don't know.